os/binfmt/binfmt_dumpmodule.c

```diff
@@ -91,8 +91,9 @@ int dump_module(FAR const struct binary_s *bin)
 		berr("  argv:      %p\n", bin->argv);
 		berr("  entrypt:   %p\n", bin->entrypt);
 		berr("  mapped:    %p size=%d\n", bin->mapped, bin->mapsize);
-		berr("  alloc:     %p %p %p\n", bin->alloc[ALLOC_TEXT], bin->alloc[ALLOC_CTOR], bin->alloc[ALLOC_DTOR]);
+		berr("  text alloc:     %p\n", bin->alloc[ALLOC_TEXT]);
```

Comment:
1. Should this be text? Not to be start of alloc?
2. don't you need to add ro and data?
These queries could be treated in another pr.

Reply:
Modified. Thanks.